Example/CollectionViewToolsExample/Main/Factories/ContentViewSectionItemsFactory.swift

```diff
@@ -5,29 +5,41 @@
 import CollectionViewTools
 
 protocol ContentViewCellItemFactoryOutput {
-    func reloadCollectionView()
-    func removeContentViewState(_ state: ContentViewState)
+    func updateContentView(for viewState: ViewState & Expandable)
+    func removeContentView(for viewState: ImageViewState)
```

Comment:
```suggestion
    func updateContentViewState(_ viewState: ViewState & Expandable)
    func removeContentViewState(_ viewState: ImageViewState)
```
What do you think of this option?

Reply:
I think that initial methods naming seems more suitable

Reply:
ok